SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -1,50 +1,66 @@
-# Get started: 2 - Get and build the GDK’s Unreal Engine Fork
+# Get started: 2 - Get and build the SpatialOS Unreal Engine fork
```

Comment:
Not 100% sure about the wording of this. Are we calling it the SpatialOS Unreal Engine fork, or the GDK for Unreal fork? etc.

Reply:
either is fine but I think SpatialOS looks cooler

Reply:
SpatialOS _does_ look cooler.

Reply:
The official term in the Canonology is: "Unreal Engine fork" .
I think we do need to use "GDK" or "SpaitialOS" when we first introduce it but we need to make sure it's consistent and not change it without reason.
"The SpatialOS Unreal Engine fork" is easier to use outside of the context of the GDK for Unreal docs because otherwise you get "the GDK for Unreal Unreal Engine fork" and that's nasty.

Reply:
Adding "SpatialOS Unreal Engine fork" to the canonology. Let me know if we change it back.